cmd/osbuild-image-tests/main_test.go

```diff
@@ -92,39 +94,27 @@ func splitExtension(path string) (string, string) {
 
 // testImageInfo runs image-info on image specified by imageImage and
 // compares the result with expected image info
-func testImageInfo(imagePath string, rawImageInfoExpected []byte) error {
+func testImageInfo(t *testing.T, imagePath string, rawImageInfoExpected []byte) {
 	var imageInfoExpected interface{}
 	err := json.Unmarshal(rawImageInfoExpected, &imageInfoExpected)
-	if err != nil {
-		return fmt.Errorf("cannot decode expected image info: %v", err)
-	}
+	require.Nil(t, err)
```

Comment:
You can add the error message to the `require.Nil` function and when that fails we'll see a humanly understandable error message instead of that something was expected to be nil and it wasn't.  For example:
```
require.Nilf(t, err, "cannot decode expected image info: %v", err)
```

Note: I personally used `.Nilf` while Brian uses just `.Nil` and both seem to be able to use string formatting for the error message.